### DISCUSSION TITLE
Background colors for Grid Rows

### DISCUSSION THREAD

**Discussion by djbbiz** [cortexrd/Knack-Toolkit-Library#272] [General]
I noticed this code in the system:
.ktlTable--rowHover tbody tr:hover {
    background-color: rgb(255, 128, 128, 0.2) !important;
    transition: background-color .2s ease-out;
}

If I want to change the color scheme do we have to over-ride with custom CSS? Was this color added for a purpose or can it be changed? I have some CSS that is conflicting and in some rows where I have white fonts that look washed out as it changed the background color to a light pink.

**Comment by cortexrd** [comment]
Dean,

Try turning off the **rowHoverHighlight** flag in **ktl.core.setCfg** function, under the section **enabled**.

![image](https://github.com/cortexrd/Knack-Toolkit-Library/assets/42851359/52fb8a12-37b3-4c0f-8684-bb7228efea15)

You can also disable **inlineEditColor**.  It could be a combination of both that cause your issue.

Norm

**Comment by djbbiz** [comment]
Worked great thanks!
